src/api/sso/src/slack-bolt-client.js

```diff
@@ -0,0 +1,60 @@
+const { logger } = require('@senecacdot/satellite');
+const { App } = require('@slack/bolt');
+
+const SLACK_BOT_TOKEN = 'slack-bot-token-placeholder';
+const SIGNING_SECRET = 'signing-secret-placeholder';
+const CHANNEL_ID = 'channel-id-placeholder';
+
+const constructMessageBlock = (displayName, githubUsername, githubAvatarUrl) => {
+  const messageBlock = [
+    {
+      type: 'header',
+      text: {
+        type: 'plain_text',
+        emoji: true,
+        text: 'New Member Notification :tada:',
+      },
+    },
+    {
+      type: 'divider',
+    },
+    {
+      type: 'context',
+      elements: [
+        {
+          type: 'image',
+          image_url: `${githubAvatarUrl}` || 'https://icons8.com/icon/23279/add-user-male',
+          alt_text: 'avatar',
+        },
+        {
+          type: 'mrkdwn',
+          text: `*${displayName}* has joined the Telescope community. Be the first one to welcome them`,
+        },
+        {
+          type: 'mrkdwn',
+          text: `<https://www.github.com/${githubUsername} | *Visit Github Profile*>`,
+        },
+      ],
+    },
+  ];
+  return messageBlock;
+};
+const publishSignUpMessage = async (displayName, githubUsername, githubAvatarUrl) => {
+  if (!displayName && !githubUsername) throw new Error('displayName and gitHubUsername required');
+  const app = new App({
+    token: SLACK_BOT_TOKEN,
+    signingSecret: SIGNING_SECRET,
+  });
```

Comment:
Maybe it'd be better to have this in a separate function, like `createSlackApp` or something like that, and it should check if `SLACKBOT_TOKEN`, `SECRET_ID` and `CHANNEL_ID` exist before creating the app.